src/components/Controls/Control/ControlOptionIcon/index.tsx

```diff
@@ -1,5 +1,6 @@
-import Icon from 'components/Icon';
-import { ControlChildrenType } from '../Control/types';
+import { contents300, secondary100 } from 'colors/v3';
+import Icon, { IconName } from 'components/Icon';
+import { ControlChildrenType } from '../types';
 
 const ICON_NAME = {
```

Comment:
```suggestion
const ICON_NAME: Record<string, Record<"on" | "off", IconName>> = {
```
이런 식으로 쓰면 밑에 as 안쓸 수 있을까요?

Reply:
오.. 넘 좋은 리뷰 감사합니다
추가적으로 string말고 ControlOption타입을 키로 받아 요렇게 더 좁혀서 사용하도록 하겠습니다!
```javascript
import type { ControlOption } from '../types';

const ICON_NAME: Record<ControlOption, Record<'on' | 'off', IconName>> = {
...
}
```